### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by fvictorio**

### CODE REVIEW

File: docs/advanced/buidler-runtime-environment.md [hunk 1]
```diff
@@ -1,70 +1,70 @@
-# Buidler Runtime Environment (BRE)
+# Hardhat Runtime Environment (HRE)
```
Comment (alcuadrado) [inline docs/advanced/buidler-runtime-environment.md:1]: File name not renamed

Comment (alcuadrado) [reply docs/advanced/buidler-runtime-environment.md:1]: btw, the file names are used as links

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 This document contains some tips on how to collaborate in this project.
 
 ## Project structure
 
 This repository is a monorepo handled with [Lerna](https://github.com/lerna/lerna).
 
-There's a folder for each subproject in `packages/`. All of them are plugins, except for `/packages/buidler-core` which
-is the main project (i.e. the one that's published as [@nomiclabs/buidler](https://npmjs.com/package/@nomiclabs/buidler)).
+There's a folder for each subproject in `packages/`. All of them are plugins, except for `/packages/hardhat-core` which
+is the main project (i.e. the one that's published as [@nomiclabs/hardhat](https://npmjs.com/package/@nomiclabs/hardhat)).
```
Comment (alcuadrado) [inline CONTRIBUTING.md:10]: `@nomiclabs/buidler` is going to be just `hardhat`

Comment (alcuadrado) [reply CONTRIBUTING.md:10]: This mistake is present in lots of places, but I'll just add this comment

File: docs/advanced/building-plugins.md [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
 
 - **Rule of thumb #2:** If your plugin P depends on another plugin P2, P2 should be a peer dependency of P, and P2's peer dependencies should be peer dependencies of P.
 
-- **Rule of thumb #3:** If you have a non-Buidler dependency that your users may `require()`, it should be a peer dependency.
+- **Rule of thumb #3:** If you have a non-Hardhat dependency that your users may `require()`, it should be a peer dependency.
 
 - **Rule of thumb #4:** Every `peerDependency` should also be a `devDependency`.
 
-Also, if you depend on a Buidler plugin written in TypeScript, you should add it's type extensions' `.d.ts` file to the `files` array of `tsconfig.json`.
+Also, if you depend on a Hardhat plugin written in TypeScript, you should add it's type extensions' `.d.ts` file to the `files` array of `tsconfig.json`.
```
Comment (alcuadrado) [inline docs/advanced/building-plugins.md:50]: TODO-HH: This is outdated now

File: docs/buidler-evm/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Buidler EVM
+# Hardhat EVM
```
Comment (alcuadrado) [inline docs/buidler-evm/README.md:1]: Hardhat Network

Comment (alcuadrado) [reply docs/buidler-evm/README.md:1]: Repeated in multiple places

File: docs/buidler-evm/README.md [hunk 5]
```diff
@@ -91,11 +91,11 @@ error message in the following cases:
 
 ## `console.log`
 
-Buidler EVM allows you to print logging messages and contract variables calling `console.log()` from your Solidity code. You can see an example in the Sample Project. Follow the steps in [Quick Start](/getting-started/#quick-start) to try it out.
+Hardhat EVM allows you to print logging messages and contract variables calling `console.log()` from your Solidity code. You can see an example in the Sample Project. Follow the steps in [Quick Start](/getting-started/#quick-start) to try it out.
 
 - You can use it in calls and transactions. It works with `view` functions, but not in `pure` ones.
 - It always works, regardless of the call or transaction failing or being successful.
-- To use it you need to import `@nomiclabs/buidler/console.sol`.
+- To use it you need to import `@nomiclabs/hardhat/console.sol`.
```
Comment (alcuadrado) [inline docs/buidler-evm/README.md:98]: Note: this will be `hardhad/console.sol`

File: docs/buidler-evm/README.md [hunk 6]
```diff
@@ -148,12 +148,12 @@ eth_call
...
+the in-process Hardhat EVM provider. See [Hardhat EVM's config](../config/README.md#hardhat-evm-network) enabled it in both.
 
-## Buidler EVM initial state
+## Hardhat EVM initial state
 
-Buidler EVM is initialized by default in this state:
+Hardhat EVM is initialized by default in this state:
 
 - A brand new blockchain, just with the genesis block.
 - 20 accounts with 10000 ETH each
```
Comment (alcuadrado) [inline docs/buidler-evm/README.md:159]: TODO-HH: The list of accounts is outdated

File: docs/config/README.md [hunk 7]
```diff
@@ -42,49 +42,49 @@ module.exports = {
...
+- `from`: The address to use as default sender. If not present the first account of the Hardhat EVM is used.
 
 - `gas`: Its value should be `"auto"` or a number. If a number is used, it will be the gas limit used by default in every transaction. If `"auto"` is used, the gas limit will be automatically estimated. Default value: `9500000`.
 
 - `gasPrice`: Its value should be `"auto"` or a number. This parameter behaves like `gas`. Default value: `8000000000`.
 
 - `gasMultiplier`: A number used to multiply the results of gas estimation to give it some slack due to the uncertainty of the estimation process. Default: `1`.
 
-- `accounts`: An array of the initial accounts that Buidler EVM will create. Each of them must be an object with `privateKey` and `balance` fields. Both of them `0x`-prefixed strings. By default, it has 20 accounts with 10000 ETH each.
+- `accounts`: An array of the initial accounts that Hardhat EVM will create. Each of them must be an object with `privateKey` and `balance` fields. Both of them `0x`-prefixed strings. By default, it has 20 accounts with 10000 ETH each.
```
Comment (alcuadrado) [inline docs/config/README.md:71]: TODO-HH: This is outdated

File: docs/getting-started/README.md [hunk 8]
```diff
@@ -1,31 +1,31 @@
...
 
-A barebones installation with no plugins allows you to create your own tasks, compile your Solidity code, run your tests and run a local development network you can deploy your contracts to (Buidler EVM).
+A barebones installation with no plugins allows you to create your own tasks, compile your Solidity code, run your tests and run a local development network you can deploy your contracts to (Hardhat EVM).
 
-To create your Buidler project run `npx buidler` in your project folder:
+To create your Hardhat project run `npx hardhat` in your project folder:
 
 ```
-$ npx buidler
+$ npx hardhat
```
Comment (alcuadrado) [inline docs/getting-started/README.md:28]: TODO-HH: This message is outdated

File: docs/guides/buidler-console.md [hunk 9]
```diff
@@ -63,8 +65,8 @@ You will also notice that the console has the handy history feature you expect o
 Interacting with the Ethereum network and your smart contracts are asynchronous operations, hence most APIs and libraries
 use JavaScript's `Promise` for returning values.   
 
-To make things easier, Buidler's console supports `await` top-level await (i.e. `console.log(await web3.eth.getBalance()`). To use this feature, you need to be using Node 10 or higher.
+To make things easier, Hardhat's console supports `await` top-level await (i.e. `console.log(await web3.eth.getBalance()`). To use this feature, you need to be using Node 10 or higher.
 
-For any help or feedback you may have, you can find us in the [Buidler Support Telegram group](http://t.me/BuidlerSupport).
+For any help or feedback you may have, you can find us in the [Hardhat Support Telegram group](http://t.me/HardhatSupport).
```
Comment (alcuadrado) [inline docs/guides/buidler-console.md:70]: TODO-HH: This is outdated. Should point to the discord server.

**Comment by alcuadrado** [comment]
Some files from the docs have buidler in their name. Others have `hardhat-evm`

File: packages/buidler-core/src/internal/cli/cli-with-a-typo.ts [hunk 10]
```diff
@@ -2,6 +2,6 @@
 /* tslint:disable */
 import chalk from "chalk";
 
-console.error(chalk.cyan("You probably meant to type buidler. We got you."));
+console.error(chalk.cyan("You probably meant to type hardhat. We got you."));
```
Comment (alcuadrado) [inline packages/buidler-core/src/internal/cli/cli-with-a-typo.ts:5]: We can delete this file. It should also be removed from the `bin` section of `package.json`.

File: packages/buidler-core/src/internal/cli/project-creation.ts [hunk 11]
```diff
@@ -53,7 +53,7 @@ async function printWelcomeMessage() {
 
   console.log(
     chalk.cyan(
-      `${emoji("👷 ")}Welcome to ${BUIDLER_NAME} v${packageJson.version}${emoji(
```
Comment (alcuadrado) [inline packages/buidler-core/src/internal/cli/project-creation.ts:56]: The message above is outdated

**Comment by alcuadrado** [comment]
There are some mentions of hardhat-evm

File: packages/buidler-core/src/internal/cli/project-creation.ts (hunk 11 above)
Comment (fvictorio) [reply packages/buidler-core/src/internal/cli/project-creation.ts:56]: You mean the ascii art banner?

Comment (alcuadrado) [reply packages/buidler-core/src/internal/cli/project-creation.ts:56]: yes

**Comment by alcuadrado** [comment]
YoLo 🤪

**Pull request merged**